### PR TITLE
Update model to use correct pluralization rules and update some deps for ng4 compatibility

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: src/app/connections/view/view.component.html [hunk 1]
```diff
@@ -1,11 +1,8 @@
-<div class="row row-cards-pf connections-view"
-     [ngClass]="{edit: 'edit', create: 'create'}[mode] || 'view'"
-     *ngIf="connection">
+<div class="row row-cards-pf connections-view" [ngClass]="{edit: 'edit', create: 'create'}[mode] || 'view'" *ngIf="connection">
```
Comment (gashcrumb) [inline src/app/connections/view/view.component.html]: Are we not doing newlines between element attributes anymore?

Comment (jimmidyson) [inline src/app/connections/view/view.component.html]: Sorry! Damn editor formatting aggressively... Reverted.

**Comment by jimmidyson** [comment]
@gashcrumb Any chance of another look so we can merge?

**Comment by gashcrumb** [comment]
Yep, sorry yesterday was a holiday here, looking now.

**Pull request merged**